assets/main.js.liquid

```diff
@@ -1,339 +1 @@
r")},bindShopClick:function(){var e=this;$(".js-scroll-to-shop").on("click",function(t){t.preventDefault(),e.scrollToShop(),$(this).trigger("blur")})},scrollToShop:function(){var e=$("#front-splash").height();$("body").stop().animate({scrollTop:e},"500","swing")}},Site.Product={$gallery:$("#product-gallery"),$galleryRow:$("#product-gallery-row"),init:function(){var e=this;e.$gallery.length&&e.$gallery.imagesLoaded(function(){e.setGalleryDimensions()}),$("#product-select").length&&e.productSelect(),$("#related-products").length&&e.pickRelated()},setGalleryDimensions:function(){var e=this,t=0,i=$(window).width();if($(".product-gallery-item").each(function(){t+=$(this)[0].getBoundingClientRect().width}),e.$galleryRow.css("transition","none"),e.$galleryRow.width(t),t>i){var o=t+(t-i);e.$gallery.width(o).css("left",i/2-o/2+"px"),e.$galleryRow.css("left",o/2-t/2+"px"),e.bindGalleryDrag()}else e.$gallery.width(t).css("left",i/2-t/2+"px"),e.$galleryRow.css({left:"0"}),e.unbindGalleryDrag()},bindGalleryDrag:function(){var e=this;e.$galleryRow.pep({axis:"x",constrainTo:"parent",place:!1,useCSSTranslation:!1}).css("cursor","ew-resize")},unbindGalleryDrag:function(){var e=this;$.pep.unbind(e.$galleryRow),e.$galleryRow.css("cursor","default")},selectCallback:function(e){e&&e.available===!0?($("#product-add-holder").removeClass("out-of-stock"),$("#add").removeAttr("disabled")):($("#product-add-holder").addClass("out-of-stock"),$("#add").attr("disabled","disabled"))},productSelect:function(){var e=this,t=JSON.parse($("#product-info").attr("data-product"));new Shopify.OptionSelectors("product-select",{product:t,onVariantSelected:e.selectCallback}),$("#product-add-holder").addClass("padding-top-small")},pickRelated:function(){$(".related-products-item").pick(4),$("#related-products").removeClass("u-hidden")}},Site.Blog={$blogItem:$(".blog-item, .blog-item-mobile, .blog-item-single"),$blogIframes:null,init:function(){var e=this;e.$blogItem.length&&(e.$blogIframes=e.$blogItem.find("iframe"),e.$blogIframes.length&&e.scaleIframes())},scaleIframes:function(){var e=this;e.$blogIframes.each(function(){var e=$(this).attr("width"),t=$(this).attr("height"),i=$(this).width(),o=t/e;$(this).css("height",i*o+"px")})}},Site.Doodle={init:function(){var e=this,t=$.keyframe.isSupported();if($("#doodle svg")[0].setAttribute("preserveAspectRatio","none"),t){var i=0;e.keyframe=1,$("#doodle svg path, #doodle svg polyline").each(function(){var t=this;setTimeout(function(){e.drawPath($(t))},i),i+=1e3})}},getPathLength:function(e){var t=e.get(0),i=t.getTotalLength();return i},drawPath:function(e){var t=this,i=t.getPathLength(e);$.keyframe.define([{name:"keyframe-"+t.keyframe,"0%":{"stroke-dashoffset":i},"50%":{"stroke-dashoffset":0},"100%":{"stroke-dashoffset":i}}]),e.css({"stroke-dasharray":i,opacity:1}).playKeyframe({name:"keyframe-"+t.keyframe,timingFunction:"linear",iterationCount:"infinite",duration:"10s"}),t.keyframe++}},jQuery(document).ready(function(){"use strict";Site.init()});
```

Comment:
I dont see any changes to the gulpfile so why does this file keep switching between min and non min?